buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -9,22 +9,24 @@ internal class YamlWriter {
       check(base.serviceAccountCredentials.isPresent) { "ServiceAccountCredentials in fladle extension not set. https://github.com/runningcode/fladle#serviceaccountcredentials" }
     }
     check(base.debugApk.isPresent) { "debugApk must be specified" }
-    check(base.instrumentationApk.isPresent xor !base.roboScript.isNullOrBlank()) {
+    check(base.sanityRobo.get() == true || (base.instrumentationApk.isPresent xor !base.roboScript.isNullOrBlank())) {
       """
      Either instrumentationApk file or roboScript file must be specified but not both.
```

Comment:
This message needs to be updated.

Reply:
I'll refactor `base.sanityRobo.get() == true` to be in separated `if` condition.